dashboard/src/components/Cards/IssuesList.tsx

```diff
@@ -13,26 +15,32 @@ import type { TIssue } from '@/types/general';
 
 interface IIssuesList {
   issues: TIssue[];
+  failedWithUnknownIssues?: number;
   title: IBaseCard['title'];
 }
 
-const IssuesList = ({ issues, title }: IIssuesList): JSX.Element => {
+const IssuesList = ({
+  issues,
+  failedWithUnknownIssues,
+  title,
+}: IIssuesList): JSX.Element => {
+  const intl = useIntl();
   const hasIssue = issues.length > 0;
 
   const titleElement = (
     <span>
       {title}
-      {hasIssue && (
+      {(hasIssue || failedWithUnknownIssues) && (
```

Comment:
just change the definition of `hasIssue` to consider `failedWithUnknownIssues`

Reply:
Done